qa/suites/rados/singleton-nomsgr/all/ceph-kvstore-tool.yaml

```diff
@@ -0,0 +1,17 @@
+roles:
+- [client.0]
```

Comment:
- - [mon.a mgr.x osd.0 osd.1 osd.2 client.0]

Reply:
OK, I will update it

Reply:
done